app/src/main/java/org/simple/clinic/home/patients/PatientsScreenController.kt

```diff
@@ -226,4 +227,10 @@ class PatientsScreenController @Inject constructor(
         .filter { it.permissionResult == RuntimePermissionResult.GRANTED }
         .map { Ui::openScanSimpleIdCardScreen }
   }
+
+  private fun openSearchResultsScreenOnScannedPassport(events: Observable<UiEvent>): Observable<UiChange> {
```

Comment:
Rename `openSearchResultsScreenOnScannedPassport` -> `openSearchResultsForScannedPassport`?
Wdyt?

Reply:
This code was added to verify that the flow of passing the scanned QR codes to `PatientsScreenController` was working fine. I'll remove this.